src/getting-started/app-client-tools-install-linux.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: document
+
+title:  "Инсталиране на StartApp Client tools под Linux"
+date:   2013-12-02 22:58:04
+header_title:  "> gem install startapp Linux"
+header_title_fontsize: big
+lead: "Как да инсталираме Ruby, Git и StartApp Client tools под Linux"
```

Comment:
StartApp Tools instead of StartApp or everywhere we need to use StartApp Client Tools